Index.html

```diff
@@ -0,0 +1,322 @@
+                </div>
+                <div class="buttonmulti">
+                  <button type="submit" class="go-buttontonic">See project</button>
+                </div>
+              </li>
+            </li>
+          </ul>
+          <ul class="flex3">
+            <li class="toni"></li>
+            <li class="square3">
+              <li class="primarytext3">
+                <h1 class="tonic1">
+                  Tonic
+                </h1>
+                <div class="cardtonic">
+                  <h2 class="CANOPY">CANOPY</h2>
+                  <img src="./images/Counter.png" alt="counter count" class="dot1" />
+                  <h2 class="BackE">Back End dev</h2>
+                  <img src="./images/Counter.png" alt="counter count" class="dot" />
+                  <h2 class="date">2015</h2>
+                </div>
+                <div clas="adayca">
+                  <p class="adayca_p">
+                    A daily selection of privately </br>
+                    personalized reads; no accounts or </br> 
+                    sign-ups required.
+                  </p>
+                </div>
+                <div class="camulti">
+                  <h3 class="cahtml">html</h3>
+                  <h3 class="cacss">css</h3>
+                  <h3 class="caJava">JavaScript</h3>
+                </div>
+                <div class="buttonca">
+                  <button type="submit" class="go-buttonca">See project</button>
+                </div>
+              </li>
+            </li>
+          </ul>
+          <ul class="flex4">
+            <li class="mult"></li>
+            <li class="square4">
+              <li class="primarytext4">
+                <h1 class="multi4">
+                  Multi-post</br>
+                  Stories
+                </h1>
+                <div class="cardtonic">
+                  <h2 class="CANOPY">CANOPY</h2>
+                  <img src="./images/Counter.png" alt="counter count" class="dot1" />
+                  <h2 class="BackE">Back End dev</h2>
+                  <img src="./images/Counter.png" alt="counter count" class="dot" />
+                  <h2 class="date">2015</h2>
+                </div>
+                <div clas="aday4">
+                  <p class="aday4_p">
+                    A daily selection of privately </br>
+                    personalized reads; no accounts or </br> 
+                    sign-ups required.
+                  </p>
+                </div>
+                <div class="bomult">
+                  <h3 class="multhtml">html</h3>
+                  <h3 class="multcss">css</h3>
+                  <h3 class="multJava">JavaScript</h3>
+                </div>
+                <div class="buttonmult">
+                  <button type="submit" class="go-buttonmult">See project</button>
+                </div>
+              </li>
+            </li>
+          </ul>
+        </nav>
+      </main>
+      <section class="contact-me">
```

Comment:
- [ ] kindly consider giving this section a border right radius as designed in Figma  after doing that consider also this:
- [ ] 1. adding the technology categories ("Languages", "Frameworks" and "Skills") in a `ul`, each of them in a `li`
1. adding  technology in each category(e.g., "JavasScript", "HTML" and "CSS")  also a list `ul` and each of them a `li`  and also consider designing the technology in each category(e.g., "JavasScript", "HTML" and "CSS") as designed on Figma
 to make your design  an exact duplicate of the given design and lists for semantic reasons check the images below for more details 
- your design 👇 
<img width="188" alt="Screen Shot 2023-06-03 at 07 23 09" src="https://github.com/jovasal/Finish-Mob-Figma/assets/60146030/cd8a6949-d630-4437-9f12-419ca5d62bd0">

- Figma design 👇 
<img width="260" alt="Screen Shot 2023-06-03 at 07 29 54" src="https://github.com/jovasal/Finish-Mob-Figma/assets/60146030/6b8a9ceb-793e-4197-afa5-8d2f0c61aeb0">

Reply:
Border radius were made. Background color is too similar and is not easy to see it. I made a change just for check this point and return it.

![image](https://github.com/jovasal/Finish-Mob-Figma/assets/35693330/e681d855-dfe4-49fb-ab6a-16a5fe959786)

Reply:
Language is ok now as described Figma:
![image](https://github.com/jovasal/Finish-Mob-Figma/assets/35693330/192dbbd3-8572-431f-92c3-3041bfe14386)

Reply:
Technologies were used as described in recommendations. Also Code were fixed and ordered.